src/Apps/Collect2/Components/SeoProductsForArtworks.tsx

```diff
@@ -29,72 +29,108 @@ export class SeoProducts extends React.Component<SeoProductsProps> {
     // here the filtering is necessary so we can re-use the artwork list shown in the page (could include
     // non-acquireable artworks) without making an extra request. Also, seller image is a required field
     // so excluding those that don't have `partner.profile.icon.url`.
-    const artworksForSeoProduct = artworks_connection.edges.filter(edge => {
-      return get(
-        edge,
-        e => e.node.is_acquireable && e.node.partner.profile.icon.url
-      )
-    })
 
-    return artworksForSeoProduct.map(({ node }, index) => {
-      const { artists, is_price_range, partner, price } = node
-      const location = partner.locations && partner.locations[0]
-      const artistsName = artists
-        ? toSentence(node.artists.map(({ name }) => name))
-        : null
-      const isInstitution = partner.type === "Institution"
+    const edges = artworks_connection && artworks_connection.edges
+    const artworksForSeoProduct =
+      edges &&
+      edges.filter(edge => {
+        return get(edge, e => {
+          const node = e && e.node !== null && e.node
 
-      return (
-        <Product
-          key={node.__id}
-          data={{
-            name: node.title,
-            image: node.image.url,
-            description: node.meta && node.meta.description,
-            url: `${APP_URL}${node.href}`,
-            brand: {
-              "@type": "Person",
-              name: artistsName,
-            },
-            ...(isInstitution
-              ? {}
-              : {
-                  category: node.category,
-                  productionDate: node.date,
-                  offers: {
-                    "@type": "Offer",
-                    price: !is_price_range
-                      ? formatCurrency(price)
-                      : {
-                          minPrice: formatCurrency(price.split("-")[0]),
-                          maxPrice: formatCurrency(price.split("-")[1]),
+          if (node) {
+            const { is_acquireable, partner } = node
+            const iconUrl =
+              partner &&
+              partner.profile &&
+              partner.profile.icon &&
+              partner.profile.icon.url
+            return is_acquireable && iconUrl
+          } else {
+            return false
+          }
+        })
+      })
```

Comment:
Why not use our `get` utility throughout? The verbosity can be reduced

Reply:
Isn’t this all already wrapped inside a `get` block, in which case you could perhaps indeed reduce the verbosity by using the `!` operator to remove `null` from these intermediate values. i.e. `partner!.profile!.icon!.url!`

Reply:
FWIW I think it’s fine to use the verbose version in places where you’re not already using `get` to wrap a larger block, no need to imo to replace it everywhere.